config/gulp/tasks/component.js

```diff
@@ -32,5 +32,7 @@ gulp.task('watch-css', function () {
 });
 
 function convertToTmpPath(file) {
-    return file.path.replace(/src\/app\//, config.tmpApp);
+	var re = new RegExp('src\' + path.sep + 'app\' + path.sep,"g");
```

Comment:
No double slash will not work for windows.
Are there are issues with running under linux?
I see that the Tavis CI build failed but I didn't get details